lib/task-data/tasks/install-ubuntu.js

```diff
@@ -14,17 +14,10 @@ module.exports = {
         installScriptUri: '{{api.templates}}/{{options.installScript}}',
         hostname: 'localhost',
         comport: 'ttyS0',
-        domain: 'rackhd',
```

Comment:
I forgot to mention this change, you can check this PR (https://github.com/RackHD/on-tasks/pull/250), it is a requirement from ODR-751